### PR TITLE
ESC for Software Engineers Launch Blog

### PR DESCRIPTION
**Pull request by lukehoban**

### CODE REVIEW

**Review by cleverguy25 — APPROVED** [review]

File: content/blog/esc-software-engineering/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+---
+
+Last October we launched [Pulumi ESC](https://www.pulumi.com/docs/esc/), a new secrets and configuration management offering from Pulumi that manages and tames secrets and configuration complexity across all of your cloud applications and infrastructure.  We’ve seen incredible adoption of ESC over the last 6 months, with thousands of developers now managing their secrets and configuration via ESC.
+
+Today, we are excited to announce the most significant collection of Pulumi ESC enhancements since the initial launch, with three major new features that bring best-in-class software engineering capabilities to bear on your Secrets and Configuration management needs.
+
+<!--more-->
+
+* __Versioning__:  Pulumi ESC now supports versioning of environments, allowing you to see and audit every change to the secrets and configuration for an environment, pin references to an environment to a specific version or version tag, and safely roll back an environment to a previous version.
```
Comment (arunkumar611) [inline content/blog/esc-software-engineering/index.md:27]: Do we also want to add links to versioning, SDK, PSP blog posts from here? Similar to how we have in Joe's post: https://www.pulumi.com/blog/infrastructure-lifecycle-management/

**Review by arunkumar611 — APPROVED** [review]
Added one cross-link comment. Otherwise LGTM

File: content/blog/esc-software-engineering/index.md (hunk 1 above)
Comment (lukehoban) [reply content/blog/esc-software-engineering/index.md:27]: Yeah - good point - I almost did in my latest edit pass - and you suggesting it as well pushes me over the edge. Will update shortly.

File: content/blog/esc-software-engineering/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+![Environment Revisions in Pulumi Console](./revisions.png)
+
+And then see the diff between any two versions of an environment:
+
+![Environment Revision Diff in Pulumi Console](./diff2.png)
+
+Revisions can also be tagged to provide semantically meaningful names to specific revisions.  These tags can be moved later on to point to new versions when you are ready to roll them out to consumers.  The latest tag automatically always points to the latest revision.
+
+![Renvironment tags in Pulumi Console](./tags.png)
```
Comment (justinvp) [inline content/blog/esc-software-engineering/index.md:66]:
```suggestion
![Revision tags in Pulumi Console](./tags.png)
```

?

**Pull request merged**